docs/_future/getting_started_future.md

```diff
@@ -0,0 +1,146 @@
+
+#### Dependencies {#setting-up-dependencies}
+
+Once the CLI is set up, make sure your machine has the most recent version of <a href="https://www.python.org/" target="_blank">Python</a> installed. You can install Python through a package manager (such as <a href="https://brew.sh/" target="_blank">Homebrew</a> for Mac) or directly from the <a href="https://www.python.org/downloads/" target="_blank">website</a>.
+
+---
+
+### Create a new app {#create-app}
+
+Before you start developing with Bolt, you'll want to create a Slack app.
+
+To create the app, you'll run the following command:
+
+```bash
+slack create my-app -t slack-samples/bolt-python-starter-template -b future
+```
+
+This command creates an app through the CLI by cloning a specified template. In this case, the template is the <a href="https://github.com/slack-samples/bolt-python-starter-template/tree/future" target="_blank">Bolt for Python Starter Template</a> on the `future` branch. This starter template includes a "Hello World" example that demonstrates how to use <a href="/bolt-python/future/concepts#built-in-functions" target="_blank">built-in</a> and <a href="/bolt-python/future/concepts#functions" target="_blank">custom</a> Functions, <a href="https://api.slack.com/future/triggers" target="_blank">Triggers</a> and <a href="/bolt-python/future/concepts#manifest-workflows" target="_blank">Workflows</a>.
+
+Once the app is successfully created, you should see a message like this:
+
+```text
+✨ my-app successfully created
+
+🧭 Explore your project's README.md for documentation and code samples, and at any time run slack help to display a list of available commands
+
+🧑‍🚀 Follow the steps below to try out your new project
+
+1️⃣  Change into your project directory with: cd my-app
+
+2️⃣  Develop locally and see changes in real-time with: slack run
+
+3️⃣  When you're ready to deploy for production use: slack deploy
+
+🔔 If you leave the workspace, you won’t be able to manage any apps you’ve deployed to it. Apps you deploy will belong to the workspace even if you leave the workspace
+```
+
+---
+
+### Set up your trigger {#setup-trigger}
+
+As mentioned, this app comes with preexisting functionality - it uses Functions, Workflows and a Trigger set up to kick things off. Let's run a command to initialize that Trigger via the CLI.
+
+First, make sure you're in the project directory in your command line: `cd my-app`
+
+Then, run the following command to create a Trigger:
+
+```bash
+slack triggers create --trigger-def "triggers/sample-trigger.json"      
+```
+
+The above command will create a trigger for the selected workspace. Make sure to select the workspace you want and that it is appended by `(dev)`. This will create a dev instance of your app. Once the trigger is successfully created, you should see an output like this:
```

Comment:
This selector feature may change in the future. Thoughts on just saying "Select the workspace you want to trigger your workflow from. Once the trigger is successfully created, you should see an output like this:"?

```suggestion
The above command will create a trigger for the selected workspace. Make sure to select the workspace you want and that it is appended by `(dev)`. This will create a dev instance of your app. Once the trigger is successfully created, you should see an output like this:
```